doc/md/declarative/declarative-hcl.mdx

```diff
@@ -0,0 +1,225 @@
+---
+id: declarative-hcl-singleschema
+title: A Quickstart Guide to Declarative Migrations with a Single Schema
```

Comment:
@rotemtam, propose a title please

Reply:
_Getting started with HCL declarative migrations for MySQL schema_
_Managing MySQL schema with Declarative migrations using HCL_

Reply:
> _Getting started with HCL declarative migrations with a MySQL schema_ 


Sounds good to me!

Reply:
sounds good